Changelog

```diff
@@ -12,8 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 (nothing)
 
 ### Changed
-
-(nothing)
+- Make .tar.gz release tarballs reproducible by zero'ing the time stamp
```

Comment:
Nit: maybe insert a blank line before this item for consistency within this file (although I think perhaps soon we should remove some of the extra blank lines, so I do not care very much).

Reply:
Indeed a blank looks better.
I'll leave it for the next comitter :smile: